declarations/Facebook.json

```diff
@@ -15,12 +15,18 @@
       "executeClientScripts": true
     },
     "Privacy Policy": {
-      "fetch": "https://www.facebook.com/about/privacy",
+      "fetch": "https://mbasic.facebook.com/privacy/policy/printable/",
       "filter": [
-        "removeReturnToTopButtons",
-        "cleanUrls"
+        "removeEmptyAnchorsLinks",
+        "removeTrackingIDs",
+        "removeLocaleFromUrls"
       ],
-      "select": "._5tkp"
+      "select": [ 
+        "div[id=root]"
```

Comment:
`#root` seems simpler to me

Reply:
And to be more consistent you could use the same selector as the ToS: `div[role=main]`